### DISCUSSION TITLE
blog-2021-02-17-minecraft-tab-menu-image

### DISCUSSION THREAD

**Discussion by giscus** [MrPowerGamerBR/PersonalWebsiteComments#8] [Fan Comments]
# blog-2021-02-17-minecraft-tab-menu-image

perfectionisperfect asked on /r/admincraft how MineAlong put an image in Minecraft's TAB list.

https://mrpowergamerbr.com/us/blog/2021-02-17-minecraft-tab-menu-image/

**Comment by JustinTheBob** [comment]
Hey thank you for posting this! Very helpful. But I downloaded that pack and it doesn't seem to work on 1.18? Did something change or they completely removed it? Thank you!

**Reply by MrPowerGamerBR** [reply]
As far as I know they haven't changed anything, my server's resource pack still uses the same method and it works fine.

**Reply by TuskarMA** [reply]
Nope, looks like it's not working since 1.18.2 and above :с

**Reply by MrPowerGamerBR** [reply]
@TuskarMA My server is running Minecraft 1.19.1 and the TAB image still works, and someone a few days ago asked me about this and they were using MC 1.18.2 and they got it working

So if it isn't working for you, you probably made a mistake somewhere, does the image show up in chat when pasting the character?

**Reply by TuskarMA** [reply]
That's strange. I downloaded your resource pack and then just copy paste \ue238 character. The result was a white square in chat :с. 

Idk, maybe Fabric 1.19 rewrite something and doesn't allow to do this

**Reply by MrPowerGamerBR** [reply]
@TuskarMA try looking at the Minecraft's log in the Launcher (you need to enable an option for the console log to stay open when running Minecraft), maybe there's a indication of what happened there :3

**Comment by iRobSch** [comment]
Hey there! We're having some issues on our server where the image unfortunately will not load. Could you potentially help us out here? Here's an image: https://i.imgur.com/WUqmMGO.png

And this is our default.json file:
    {
            "type": "bitmap",
            "file": "minecraft:pics/logo.png",
            "height": 106,
            "ascent": 66,
            "chars": [
                "鳥"
            ]
        }

**Reply by MrPowerGamerBR** [reply]
> "file": "minecraft:pics/logo.png",

Remove the `.png`

**Comment by Zynph** [comment]
How do i get rid of the white stitching? its like white lines overlayed over my icon

**Reply by MrPowerGamerBR** [reply]
Can you share a screenshot of what you are talking about? I never experienced an issue like that

**Reply by Zynph** [reply]
https://i.imgur.com/pNIdDFu.png

**Reply by Zynph** [reply]
image is solid so idk why its happening

**Reply by Zynph** [reply]
Any idea?

**Reply by MrPowerGamerBR** [reply]
Sadly I don't have any idea, could you share the resource pack here + what character you used so I could check it on my computer?

**Reply by Zynph** [reply]
Yeh of course give me a couple mins

**Reply by Zynph** [reply]
https://www.mediafire.com/file/1644iqyvtu4axlw/snowsmp.zip/file

**Reply by Zynph** [reply]
there u go

**Reply by Zynph** [reply]
also character is the one you used in the tutorial

**Reply by MrPowerGamerBR** [reply]
Weird, I enabled the resource pack and I also have the same issue

I'm trying to figure out why this happens, but I can't figure it out, this is so weird

**Reply by Zynph** [reply]
My only though would be the size of the image or the size defined in the json?
I honestly have no clue

**Reply by MrPowerGamerBR** [reply]
@Zynph it is the `shaders` in your resource pack, they are affecting the font image ;)

**Reply by Zynph** [reply]
Ah okay so deleting that should fix it?

**Reply by MrPowerGamerBR** [reply]
Yes, I tried deleting it and it fixed the issue

**Reply by Zynph** [reply]
Deleted "shaders" but still getting it?

**Reply by MrPowerGamerBR** [reply]
You need to delete it from the `assets/minecraft/shaders` path, not only from the `assets/realms/shaders` path.

Also check if you don't have any other resource packs enabled.

**Reply by Zynph** [reply]
Just noticed as ive seen your reply, sorry. Yeh i had another pack on underneath like a total genius ;/

**Reply by Zynph** [reply]
Thank you for you help! And in such quick time as well!

**Comment by IjsbeerrM** [comment]
Is this only working with optifine? Can't get it working without. Is there a way to do it without?

**Comment by nuttapillar** [comment]
For anyone who is stuck, I think it doesn't like you using uppercase or spaces in the filenames.

**Comment by nuttapillar** [comment]
Does anyone know how to convert this to work in a Bedrock resource pack?

**Comment by F-alling** [comment]
The pack is down, can you please post  a new link?
